Makefile

```diff
@@ -2,7 +2,6 @@
 # This is the code that we want to run tests for and lint, staticcheck, etc.
 ALL_SRC := $(shell find . -name '*.go' \
 							-not -path './testbed/*' \
-							-not -path '*/third_party/*' \
```

Comment:
I had made any `third_party` directories get ignored (here and in `.golangci.yaml`) when I originally added this PDH code. Not sure if I should remove this or keep it?